internals/secrethub/audit.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 var (
-	errNoSuchFormat = func(format string) error { return errors.New("invalid format: " + format) }
+	errNoSuchFormat = func(format string) error { return errors.New("invalid format: " + format + " (audit.invalid_format)") }
```

Comment:
Is there a reason not to use `errio` here? I know it's old and ugly, but doing this concatenation manually is going to give us some headaches in the future if we're ever moving away from that package.

Reply:
In other PRs I was told to not use `errio` as we're trying to move away from it. (Admittedly, the context was a bit different.). It would be nice to have this written down somewhere so that we keep the code consistent.

Reply:
And I agree, I'll just use errio.